cv32e40p/sim/uvmt/Makefile

```diff
@@ -52,5 +52,18 @@ DPI_DASM_SPIKE_REPO  ?= https://github.com/riscv/riscv-isa-sim.git
 CV_SW_TOOLCHAIN      ?= /opt/riscv
 CV_SW_PREFIX         ?= " "
 
+YOUR_GCC_VERSION     := $(shell $(CV_SW_TOOLCHAIN)/bin/$(CV_SW_PREFIX)gcc -dumpversion)
+REQD_GCC_VERSION     := 13.0.0
+
+ifeq ($(REQD_GCC_VERSION), $(YOUR_GCC_VERSION))
```

Comment:
I don't know "gnu make" well enough to comment on the `ifeq` code. But the comments seemed to make sense.

Reply:
Yeah, I put the comments there as much for myself as others.  :rofl: